src/index.d.ts

```diff
@@ -12,6 +12,7 @@ declare module 'react-progressive-graceful-image' {
     noLazyLoad?: boolean;
     rootMargin?: string;
     threshold?: Array;
+    children?: (src: string, loading: boolean) => React.ReactNode;
```

Comment:
Thanks for PR.
Pls add `srcSetData` type too

Reply:
Sure thing, @sanishkr. Done :)